DigitalLearningSolutions.Data.Tests/Services/LoginServiceTests.cs

```diff
@@ -241,18 +241,67 @@ public void
         }
 
         [Test]
+        [TestCase(true, true, true, true, true, true, false)]
+        [TestCase(true, true, false, true, true, true, false)]
+        [TestCase(true, false, false, true, true, true, false)]
+        [TestCase(true, true, true, true, false, true, false)]
+        [TestCase(true, true, true, true, true, false, false)]
+        [TestCase(true, true, true, true, false, false, false)]
+        [TestCase(true, true, true, false, false, false, false)]
+        [TestCase(true, false, false, true, false, true, true)]
+        [TestCase(true, false, false, true, true, false, true)]
+        [TestCase(true, false, false, true, false, false, true)]
+        [TestCase(true, false, false, false, false, false, true)]
+        [TestCase(false, true, true, true, true, true, true)]
+        [TestCase(false, true, false, true, true, true, true)]
+        [TestCase(false, false, false, true, true, true, true)]
+        [TestCase(false, true, true, true, false, true, true)]
+        [TestCase(false, true, true, true, true, false, true)]
+        [TestCase(false, true, true, true, false, false, true)]
+        [TestCase(false, true, true, false, false, false, true)]
+        [TestCase(false, false, false, true, false, true, true)]
+        [TestCase(false, false, false, true, true, false, true)]
+        [TestCase(false, false, false, true, false, false, true)]
+        [TestCase(false, false, false, false, false, false, true)]
         public void
-            AttemptLogin_returns_choose_a_centre_when_logging_into_single_inactive_centre_and_successfully_logging_in()
+            AttemptLogin_returns_choose_a_centre_when_logging_into_single_inactive_centre_or_when_neither_admin_is_active_nor_delegate_account_is_active_and_approved(

```

Comment:
This test only covers single-centre users. Could we tweak the name?
"AttemptLogin_returns_choose_a_centre_for_single-centre_user_when_centre_inactive_or_when_neither_admin_is_active_nor_delegate_account_is_active_and_approved"